models/kernels/Nonlinear_Reservoir.hpp

```diff
@@ -1,3 +1,6 @@
+#ifndef NONLINEAR_RESERVOIR_HPP
```

Comment:
I think this was merged in a prior PR so this is causing a merge conflict.

Reply:
@robertbartel can you check on this conflict then I'll merge

Reply:
I've done some rebasing to get the exponential reservoir outlet changes incorporated, which should have also taken care of this.